src/BinDeps.jl

```diff
@@ -47,7 +47,7 @@ module BinDeps
     function download_cmd(url::AbstractString, filename::AbstractString)
         global downloadcmd
         if downloadcmd === nothing
-            for download_engine in @windows? (:powershell, :curl, :wget, :fetch) : (:curl, :wget, :fetch)
+            for download_engine in (is_windows() ? (:powershell, :curl, :wget, :fetch) : (:curl, :wget, :fetch))
```

Comment:
The latter, I think:
```
julia> for a in is_windows() ? [1 2] : [3 4]
ERROR: syntax: space before "[" not allowed in "? ["
 in eval(::Module, ::Any) at .\boot.jl:225
 in macro expansion at .\REPL.jl:92 [inlined]
 in (::Base.REPL.##1#2{Base.REPL.REPLBackend})() at .\event.jl:46

julia> for a in is_windows() ?[1 2] : [3 4]
       println(a)
       end
ERROR: UndefVarError: ? not defined
 in macro expansion; at .\REPL[9]:1 [inlined]
 in anonymous at .\<missing>:?
 in eval(::Module, ::Any) at .\boot.jl:225
 in macro expansion at .\REPL.jl:92 [inlined]
 in (::Base.REPL.##1#2{Base.REPL.REPLBackend})() at .\event.jl:46

julia> Base.operator_precedence(:in)
6

julia> Base.operator_precedence(:?)
2
```
So this seems like correct, albeit kinda confusing on first glance.